source/renderer/sidebar/tree-item.vue

```diff
@@ -95,8 +95,8 @@
 
 <script>
 // Tree View item component
-const findObject = require('../../source/common/util/find-object.js')
-const { trans } = require('../../source/common/lang/i18n')
+const findObject = require('../../common/util/find-object.js')
+const { trans } = require('../../common/lang/i18n')
```

Comment:
IMHO, it should be not break anything. However, I would consider doing the `require` -> `import` migration in a separate pull request by adding the [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import). It might come with some autofixing capabilities.